src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -266,7 +266,11 @@ private void initialize() {
                             .map(this::createContextMenuForGroup)
                             .orElse((ContextMenu) null));
             row.addEventFilter(MouseEvent.MOUSE_PRESSED, event -> {
-                if (event.getButton() == MouseButton.SECONDARY || event.getTarget() instanceof StackPane) {
+                if (event.getTarget() instanceof StackPane) {
```

Comment:
Use pattern matching to avoid unnecessary casts

if (event.getTarget() instanceof StackPane pane) {